Promise.all([promise1, promise2, promise3])
//     .then((results) => console.log("All promises resolved:", results))
//     .catch((error) => console.error("One of the promises rejected:", error));